db/migrations/20220109181508_session.sql

```diff
@@ -0,0 +1,20 @@
+-- +goose Up
+CREATE TABLE IF NOT EXISTS public.session
+(
+    "session_ID" integer NOT NULL GENERATED ALWAYS AS IDENTITY ( INCREMENT 1 START 1 MINVALUE 1 MAXVALUE 2147483647 CACHE 1 ),
+    "hall_ID" integer NOT NULL,
+    "movie_ID" integer NOT NULL,
+    "time" time with time zone NOT NULL,
+    CONSTRAINT "Session_pkey" PRIMARY KEY ("session_ID"),
+    CONSTRAINT "FK_<Session>_<Hall>" FOREIGN KEY ("hall_ID")
```

Comment:
```suggestion
    CONSTRAINT "FK_sessions_to_halls" FOREIGN KEY ("hall_ID")
```
In case u want to use named constraints